addons/account/models/account_journal.py

```diff
@@ -816,9 +814,9 @@ def write(self, vals):
                 journal.default_account_id.currency_id = journal.currency_id
 
         # Create the bank_account_id if necessary
-        if 'bank_acc_number' in vals:
+        if 'bank_account_number' in vals:
             for journal in self.filtered(lambda r: r.type == 'bank' and not r.bank_account_id):
-                journal.set_bank_account(vals.get('bank_acc_number'), vals.get('bank_id'))
+                journal.set_bank_account(vals.get('bank_account_number'), vals.get('bank_bic'))
```

Comment:
Is this code even used?
in what situation do we get the `bank_account_number` field in the write values?